exonum-java-binding/cryptocurrency-demo/pom.xml

```diff
@@ -224,6 +225,7 @@
           <systemPropertyVariables>
             <it.artifactFilename>${artifactFilename}.jar</it.artifactFilename>
             <it.artifactId>${artifactName}</it.artifactId>
+            <it.artifactVersion>${artifactVersion}</it.artifactVersion>
```

Comment:
Why is version added? It does not seem to be needed — you must be able to take it from ^